### ISSUE TITLE
example for iterating over loaded data gives error when trying to access results

### ISSUE THREAD

**Issue by awakenting** [SmokinCaterpillar/pypet#49]
Hey,
when I try this example code: http://pypet.readthedocs.io/en/latest/cookbook/trajectory.html#interaction-with-trajectories-after-an-experiment, I get an error when trying to access z. For that, I simply added the follow lines at the end of the 'example_01_first_steps.py' script:
```python
for run_name in traj.f_get_run_names():
    traj.f_set_crun(run_name)
    print(traj.z)
```
 The error is the following:

```
Traceback (most recent call last):
  File ".../.local/lib/python3.4/site-packages/pypet/naturalnaming.py", line 2322, in _get
    try_auto_load_directly2)
  File ".../.local/lib/python3.4/site-packages/pypet/naturalnaming.py", line 2397, in _perform_get
    result, depth = self._search(result, key, max_depth, with_links, crun)
  File ".../.local/lib/python3.4/site-packages/pypet/naturalnaming.py", line 2045, in _search
    child.v_full_name))
pypet.pypetexceptions.NotUniqueNodeError: Node `z` has been found more than once within the same depth 4. Full name of first occurrence is `results.runs.run_00000011.z` and of second `results.runs.run_00000008.z`

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File ".../example_01_first_steps.py", line 68, in <module>
    print(traj.z)
  File ".../.local/lib/python3.4/site-packages/pypet/naturalnaming.py", line 2988, in __getattr__
    with_links=self.v_root.v_with_links)
  File ".../.local/lib/python3.4/site-packages/pypet/naturalnaming.py", line 2326, in _get
    raise wildcard_exception
  File ".../.local/lib/python3.4/site-packages/pypet/naturalnaming.py", line 2311, in _get
    try_auto_load_directly1)
  File ".../.local/lib/python3.4/site-packages/pypet/naturalnaming.py", line 2397, in _perform_get
    result, depth = self._search(result, key, max_depth, with_links, crun)
  File ".../.local/lib/python3.4/site-packages/pypet/naturalnaming.py", line 2045, in _search
    child.v_full_name))
pypet.pypetexceptions.NotUniqueNodeError: Node `z` has been found more than once within the same depth 4. Full name of first occurrence is `results.runs.run_00000011.z` and of second `results.runs.run_00000008.z`

Process finished with exit code 1
```

When I instead write `print(traj.crun.z)` it works fine, so I guess the fixing of the run index only works for parameters ?

Oh and also the print statement in the example code is still in python 2.

**Comment by SmokinCaterpillar** [comment]
Yes, fixing the run index only works with parameters. A parameter is a single node in the tree containing all values (because they are known before the parameter exploration), whereas a result of each run is a new node in the trajectory tree. 

Regarding the wrong print statement, I am always happy to take pull requests :-)

**Comment by SmokinCaterpillar** [comment]
docs have been fixed in 0.4.1.
